docs/developer-guide/nodejs-api.md

```diff
@@ -224,7 +224,7 @@ The return value is an object containing the results of the linting operation. H
             output: "foo;",
             messages: [
                 {
-                    fatal: false,
+                    fatal: false, // Note that this is _only_ included if parsing failed
```

Comment:
We never include `fatal: false`. It might be better to remove this and instead include a second example that shows a fatal message with `fatal: true` and explains that we add that only for fatal messages.